src/KeywordExtractor/KeywordExtractor.php

```diff
@@ -64,6 +64,54 @@ public function run(string $string, string $sortBy = '', string $sortDir = Sorte
         return $this->keywords;
     }
 
+    public function getKeywords(): array
+    {
+        return $this->retrieveKeywords();
```

Comment:
No, the call is not necessary. On the other hand, this method is going to change (because this is a draft) and these cases I prefer using something like this, because use can see the logic of how the things are done in the method.